TestHelper.psm1

```diff
@@ -293,7 +293,7 @@ function Initialize-TestEnvironment
         [String] $TestType
     )
     
-    Write-Verbose -Verbose (`
+    Write-Verbose -Message (`
         'Initializing Test Environment for {0} testing of {1} in module {2}.' `
```

Comment:
I think this should be displayed as well, so that we know straight away that given test uses our common function for initialization.